ccd-definition/CaseEventToFields/response/partaccept/DefendantBankAccounts.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "DEFENDANT_RESPONSE_SPEC",
+    "CaseFieldID": "minusBeforeOverdrawnAmount",
+    "PageID": "DefendantBankAccounts",
+    "PageDisplayOrder": 11,
+    "PageFieldDisplayOrder": 1,
```

Comment:
There is already a DefendantResponseAdmitPartLRspec file. Please can you move these to there so that all AdmitPart journey screens appear there.